demos/using-dev-build/package.json

```diff
@@ -2,31 +2,31 @@
   "name": "@dfinity/internet-identity-e2e-tests",
   "version": "1.0.0",
   "private": true,
+  "type": "module",
   "scripts": {
     "wdio": "wdio run ./wdio.conf.ts",
-    "test": "./test.ts",
+    "opts": "NODE_OPTIONS='--loader ts-node/esm --experimental-specifier-resolution=node' \"$@\"",
```

Comment:
Is this being used by the scripts?

Reply:
Ah no good catch, I tried it out during development to solve a problem I had. Removed.